ui/public/index.html

```diff
@@ -21,6 +21,8 @@
       };
     </script>
     <% } %>
+    <script>try{new EventTarget}catch(e){document.write('<script src="https://unpkg.com/@ungap/event-target@0.1.0/min.js"><\x2fscript>')}</script>
+    <link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet">
```

Comment:
Any way we can avoid a separate set of icons? By re-implementing a set of components for shaperone for example?

Reply:
Right now the material design rendering code hard-codes material icons (via `<mwc-icon>`) element.

Yes, It could be part of a refactoring I want in how the rendering is composed to have small pieces of the UI replaced